Dockerfile

```diff
@@ -13,7 +13,7 @@ RUN DEBIAN_FRONTEND=noninteractive apt-get -y full-upgrade
 RUN DEBIAN_FRONTEND=noninteractive apt-get -y install software-properties-common git make ccache python curl bzip2
 RUN DEBIAN_FRONTEND=noninteractive add-apt-repository -y ppa:team-gcc-arm-embedded/ppa
 RUN DEBIAN_FRONTEND=noninteractive apt-get -y update
-RUN DEBIAN_FRONTEND=noninteractive apt-get -y install gcc-arm-embedded
+RUN DEBIAN_FRONTEND=noninteractive apt-get -y install gcc-arm-embedded gcc
```

Comment:
Can you please move this one up into the first instance of `apt-get -y install` line? The second instance is only there because we need to first install `software-properties` in order to be able to add the PPA, and then we install `gcc-arm-embedded` from the PPA.